validity_rollups_on_bitcoin.md

```diff
@@ -26,11 +62,11 @@ The first cryptocurrency to deploy zk proof technology to mainnet was Firo (fka
+Network optimization scaling techniques have two main goals: one is to reduce block propagation latency, and the other is to reduce the bandwidth cost of participating as a full node on the bitcoin network.[14] Reducing block propagation latency has the benefit of improving the certainty of confirmations by reducing the number of orphaned blocks, as well as making mining more "fair" by removing the advantage of faster bandwidth speeds. This helps with scalability because if bigger blocks can be relayed faster then they have less of a negative effect on miner centralization. An example of this technique is FIBRE, a protocol developed by Matt Corallo that can bring block propagation time down to a few milliseconds slower than the speed of light.[15] Reducing the cost of participating as a full node in the network has the benefit of making it cheaper and easier to run a full node, improving the decentralization of the network. This helps with scalability by making it possible to increase transaction capacity without blowing up bandwidth costs. An example of this technique is BIP-152, also known as "compact block relay", which was also developed by Matt Corallo based on earlier work by Greg Maxwell.[16] BIP-152 reduces the amount of data that needs to be sent when propagating blocks through the bitcoin peer-to-peer network, resulting in reduced bandwidth costs for full nodes.
 
 Sharding is a technique where a cryptocurrency's block processing and storage is split into two or more groups of nodes. Security is shared between shards so that no shard is easier to attack than any other shard. The effect of sharding is that full nodes can be confident that all of the cryptocurrency's consensus rules are being followed while only needing to store and execute a fraction of all of the transactions occurring on the network. The more shards there are, the more throughput can be supported without increasing the computational burden on any given full node. Vitalik Buterin first proposed sharding as a means of scaling the Ethereum protocol in a blog post published in October 2014.[13] The first implementation of a sharded blockchain was Zilliqa, which went live in January 2019.[14] Ethereum developers are still planning to implement sharding, although the specifics of how it will work have changed over time.[15]
```

Comment:
> Security is shared between shards so that no shard is easier to attack than any other shard.

Seems categorically wrong, each shard has their own security and it is not shared.  It is parameterized the same, but the shard-specific circumstances are unique. One shard can be entirely Sybil'd, for instance.

Reply:
I'm trying to summarize the security properties described here:

> Sharding is a technique that gets you all three. A sharded blockchain is...
> 
> Secure: an attacker can't target a small part of the system with a small amount of resources; they can only try to dominate and attack the whole thing...
> 
> The random sampling prevents the attacker from concentrating their power on one shard. In a 100-chain multichain ecosystem, the attacker only needs ~0.5% of the total stake to wreak havoc: they can focus on 51% attacking a single chain. In a sharded blockchain, the attacker must have close to ~30-40% of the entire stake to do the same (in other words, the chain has shared security). Certainly, they can wait until they get lucky and get 51% in a single shard by random chance despite having less than 50% of the total stake, but this gets exponentially harder for attackers that have much less than 51%. If an attacker has less than ~30%, it's virtually impossible.

https://vitalik.ca/general/2021/04/07/sharding.html

However later in that post, Vitalik also says:

> Sharded blockchains depend on sharded peer-to-peer networks, and each individual p2p "subnet" is easier to attack because it has fewer nodes. The [subnet model used for data availability sampling](https://hackmd.io/@vbuterin/sharding_proposal#Blob-publication-process) mitigates this because there is some redundancy between subnets, but even still there is a risk.

Is that what you're describing when you say "One shard can be entirely Sybil'd"?

Reply:
It's not possible to choose the shard one validates on so as long as there's a sufficient total number of validator accounts it's statically unlikely that any malicious subset with less than 1/3 of the total stake will end up with >1/3 on any one shard.  And all that would accomplish is preventing finality, it doesn't allow reversal of finalized shard blocks, which requires 2/3.

Reply:
@delbonis  thanks for that. So would you say my initial characterization was accurate? Or I should remove this sentence since it doesn't apply to all kinds of sharding?

> Security is shared between shards so that no shard is easier to attack than any other shard.